includes/Services/LabkiPackManager.php

```diff
@@ -84,7 +84,7 @@ public function validatePackDependencies( ContentRefId $refId, array $packsToIns
 		$manifestPath = $worktreePath . '/manifest.yml';
 		$manifestContent = file_get_contents( $manifestPath );
 		$manifest = Yaml::parse( $manifestContent );
-		$packs = $manifest['packs'];
+		$packs = $manifest['packs'] ?? [];
```

Comment:
`packs` is a required key in the schema: https://github.com/Aharoni-Lab/labki-packs-tools/blob/4a0665e1047d469f9de6a6a98c0cd104e3590153/schema/v1_0_0/manifest.schema.json#L4